en-US/Grammar.xml

```diff
@@ -606,11 +872,56 @@ Split contractions and abbreviations into separate paragraphs. -->
 		</formalpara>
 
 	</section> 
-	 <section id="gender-references">
-		<title>Gender References</title>
+	 <section id="pronouns-gender-references">
+		<title>Pronouns and Gender References</title>
+		 <para>
+			Do not use gender-specific pronouns in documentation, except to refer to a specific named user, such as in a case study. It is far less awkward to read a sentence that uses "they" and "their" rather than "he/she" and "his/hers". It is acceptable to use "they" to refer to one person, with a plural verb. 
```

Comment:
Instead of "It is far less awkward to read..." how about "It is easier to read..."

Reply:
Fixed